zips/zip-0230.rst

```diff
@@ -315,23 +315,48 @@ An issuance action, ``IssueAction``, is the instance of issuing a specific Custo
 +-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
 | Bytes                       | Name                     | Data Type                                 | Description                                                         |
 +=============================+==========================+===========================================+=====================================================================+
-|``2``                        |``assetDescSize``         |``byte``                                   |The length of the asset description string in bytes.                 |
+|``varies``                   |``assetDescSize``         |``compactSize``                            |The length of the asset description string in bytes.                 |
```

Comment:
The cost of storing the asset description string needs to be taken into account when deciding on the fee for:
* the transaction that creates a new asset (which requires the asset description to be stored in the global state);
* each issuance transaction.

It may be useful to limit the size of ``assetDescSize`` by consensus rule.

Reply:
We have an explicit consensus rule in [L350](https://github.com/QED-it/zips/blob/20ef938ec2f3cd6cb83719217b66196fa46c95e6/zips/zip-0227.rst?plain=1#L350) to limit the value of `assetDescSize` (and then to confirm the asset description string is those number of bytes)